src/lib/layouts/workflow-run-layout.svelte

```diff
@@ -133,32 +144,45 @@
   });
 </script>
 
-<LabsModeGuard>
-  <div
-    class="absolute bottom-0 left-0 right-0 {$viewDataEncoderSettings
-      ? 'top-[540px]'
-      : 'top-0'}
+{#if raw}
+  <div class="relative h-auto whitespace-break-spaces rounded bg-primary p-4">
+    <CopyButton
+      copyIconTitle="Copy JSON to clipboard"
+      copySuccessIconTitle="JSON copied to clipboard"
```

Comment:
We could also just use the existing copy for copy icons here if you wanted

```suggestion
      copyIconTitle={translate('common.copy-icon-title')}
      copySuccessIconTitle={translate('common.copy-success-icon-title')}
```

Reply:
👍